src/main/java/framework/DriverManager.java

```diff
@@ -26,13 +31,31 @@ public static DriverManager getInstance() {
         return instance;
     }
 
-    private void initializeDriver() {
-        driver = new FirefoxDriver();
+    private void initializeDriver(){
         String baseUrl = "https://login.salesforce.com";
+
+        if(browser.equalsIgnoreCase("Firefox")){
+            driver = new FirefoxDriver();
+        }
+        else if(browser.equalsIgnoreCase("Chrome")){
+            System.setProperty("webdriver.chrome.driver", "drivers/chromedriver.exe");
+            driver = new ChromeDriver();
+        }
+        else if(browser.equalsIgnoreCase("IE")){
+            System.setProperty("webdriver.ie.driver", "drivers/IEDriverServer.exe");
+            driver = new InternetExplorerDriver();
+        }
+        else if(browser.equalsIgnoreCase("Safari")){
+            driver = new SafariDriver();
+        }
+
         driver.manage().timeouts().implicitlyWait(15, TimeUnit.SECONDS);
-        wait = new WebDriverWait(driver, 15);
+        wait = new WebDriverWait(driver,15);
```

Comment:
Consider using constants for timeouts
    public static final int TIMEOUT_MIN = 2;
    public static final int TIMEOUT_NORMAL = 15;